shopfloor/actions/message.py

```diff
@@ -125,6 +125,12 @@ def line_not_available_in_picking(self, picking):
             ),
         }
 
+    def line_assigned_to_another_user(self):
+        return {
+            "message_type": "error",
+            "body": _("A line is being processed by another user."),
```

Comment:
Propagate the error message raised

Reply:
Do you mean, change this line https://github.com/OCA/wms/pull/536/files#diff-81fce66279454c4c0be2adf1f46eb4e629c63edfe409c4cf6bda95940686f9bcR866 to not have this message but the text from the exception raised ?

Reply:
yes

Reply:
Done